LocalSettings.php

```diff
@@ -5350,7 +5350,7 @@
 		'godaigowiki' => 'monobook',
 		'ofthevampirewiki' => 'dusktodawn',
 		'ontariobrasswiki' => 'monobook',
-		'pculsdwiki' => 'monobook',
+		'pculsdwiki' => 'Timeless',
```

Comment:
you need to enable Timeless in wmgUseTimeless (if not already done) and please keep timeless without a capital. Also no need to open new PRs you can just edit this one.